api.go

```diff
@@ -70,33 +70,40 @@ type ClientPersistent struct {
 	client        *Client
 }
 
-// GetVersion returns version
-func (c *Client) GetVersion() (string, error) {
-	resp, err := c.api.DoAndGetResponseBody(
+func (c *Client) GetVersion() (ver string, err error) {
```

Comment:
Do not remove the comment section above any global function or method in Go, as this will result in a linter warning.